test/contract/EasyAuction.spec.ts

```diff
@@ -1434,8 +1466,111 @@ describe("EasyAuction", async () => {
       const auctionData = await easyAuction.auctionData(auctionId);
       expect(auctionData.volumeClearingPriceOrder).to.equal(0);
     });
+    it("checks whether the minimalFundingThreshold is not met", async () => {
+      const initialAuctionOrder = {
+        sellAmount: ethers.utils.parseEther("10"),
+        buyAmount: ethers.utils.parseEther("10"),
+        userId: BigNumber.from(0),
+      };
+      const sellOrders = [
+        {
+          sellAmount: ethers.utils.parseEther("1"),
+          buyAmount: ethers.utils.parseEther("1").div(2),
+          userId: BigNumber.from(0),
+        },
+        {
+          sellAmount: ethers.utils.parseEther("1"),
+          buyAmount: ethers.utils.parseEther("1").div(4),
+          userId: BigNumber.from(0),
+        },
+      ];
+      const {
+        auctioningToken,
+        biddingToken,
+      } = await createTokensAndMintAndApprove(
+        easyAuction,
+        [user_1, user_2],
+        hre,
+      );
+
+      const auctionId: BigNumber = await sendTxAndGetReturnValue(
+        easyAuction,
+        "initiateAuction(address,address,uint256,uint256,uint96,uint96,uint256,uint256)",
+        auctioningToken.address,
+        biddingToken.address,
+        60 * 60,
+        60 * 60,
+        initialAuctionOrder.sellAmount,
+        initialAuctionOrder.buyAmount,
+        1,
+        ethers.utils.parseEther("5"),
+      );
+      await placeOrders(easyAuction, sellOrders, auctionId, hre);
+
+      await closeAuction(easyAuction, auctionId);
+      const price = await calculateClearingPrice(easyAuction, auctionId);
+
+      await easyAuction.verifyPrice(auctionId, encodeOrder(price));
+      expect(price).to.eql(initialAuctionOrder);
+      const auctionData = await easyAuction.auctionData(auctionId);
+      expect(auctionData.minFundingThresholdNotReached).to.equal(true);
+    });
   });
   describe("claimFromAuctioneerOrder", async () => {
+    it("checks that auctioneer receives all their auctioningTokens back if minFundingThreshold was not met", async () => {
+      const initialAuctionOrder = {
+        sellAmount: ethers.utils.parseEther("10"),
+        buyAmount: ethers.utils.parseEther("10"),
+        userId: BigNumber.from(0),
+      };
+      const sellOrders = [
+        {
+          sellAmount: ethers.utils.parseEther("1"),
+          buyAmount: ethers.utils.parseEther("1").div(2),
+          userId: BigNumber.from(0),
+        },
+        {
+          sellAmount: ethers.utils.parseEther("1"),
+          buyAmount: ethers.utils.parseEther("1").div(4),
+          userId: BigNumber.from(0),
+        },
+      ];
```

Comment:
The tests should use different users from the auction creator for the orders, just to be safe that this does not influence the final balances.

```suggestion
      const sellOrders = [
        {
          sellAmount: ethers.utils.parseEther("1"),
          buyAmount: ethers.utils.parseEther("1").div(2),
          userId: BigNumber.from(1),
        },
        {
          sellAmount: ethers.utils.parseEther("1"),
          buyAmount: ethers.utils.parseEther("1").div(4),
          userId: BigNumber.from(2),
        },
      ];
```